test_my_idea.py

```
@@ -0,0 +1,86 @@
+        
+        # Show QADI thinking process summary
+        print(f"\n🔬 QADI THINKING PROCESS:")
+        print("-" * 50)
+        for phase, phase_result in result.phases.items():
+            if phase_result:
+                print(f"• {phase.upper()}: {len(phase_result.generated_ideas)} insights")
+        
+        print(f"\n🎯 Analysis complete! The system used systematic thinking")
+        print(f"   ({' → '.join(result.phases.keys())}) to generate answers.")
+        
+    except Exception as e:
```

Ruff (0.11.9)

Do not catch blind exception: `Exception`

(BLE001)